app/test_main.py

```diff
@@ -1,3 +1,23 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+class TestGetHumanAge:
+
+    @pytest.mark.parametrize("cat_age, dog_age, result",
+                             [(0, 0, [0, 0]),
+                              (14, 14, [0, 0]),
+                              (15, 15, [1, 1]),
+                              (24, 24, [2, 2]),
+                              (-1, -1, [0, 0])
+                              ])
```

Comment:
fix the identation here

Reply:
added pytest.param if wrong idk what to fix )